src/TLuaInterpreter.cpp

```diff
@@ -11955,6 +11955,55 @@ int TLuaInterpreter::sendIrc( lua_State * L )
     return 0;
 }
 
+int TLuaInterpreter::setServerEncoding(lua_State * L)
```

Comment:
Why `server` - might as well be `getEncoding` and `setEncoding`, it's not like we'll have a client encoding - that's bundled in the same setting.